spec/grid_view_spec.rb

```diff
@@ -3,7 +3,8 @@
 RSpec.describe GridView do
   it "renders a grid to output" do
     output = StringIO.new
-    view = described_class.new(output: output)
+    grid = {"B2" => "X", "C3" => "O"}
```

Comment:
It doesn't feel great here to pass a hash of data. This exposed data feels a bit like seeing the sausage being made.